src/test/resources/config/test-it-config.yaml

```
@@ -73,6 +73,7 @@ notify:
   liveAccountCreatedEmailTemplateId: ${NOTIFY_LIVE_ACCOUNT_CREATED_EMAIL_TEMPLATE_ID:-pay-notify-live-account-created-email-template-id}
   stripeDisputeCreatedEmailTemplateId: ${NOTIFY_STRIPE_DISPUTE_CREATED_EMAIL_TEMPLATE_ID:-pay-notify-stripe-dispute-created-email-template-id}
   stripeDisputeLostEmailTemplateId: ${NOTIFY_STRIPE_DISPUTE_LOST_EMAIL_TEMPLATE_ID:-pay-notify-stripe-dispute-lost-email-template-id}
+stripeDisputeLostAndServiceChargedEmailTemplateId: ${NOTIFY_STRIPE_DISPUTE_LOST_AND_SERVICE_CHARGED_EMAIL_TEMPLATE_ID:-pay-notify-stripe-dispute-lost-email-template-id}
```

Integration tests could be failing due to indentation here. 
```suggestion
  stripeDisputeLostAndServiceChargedEmailTemplateId: ${NOTIFY_STRIPE_DISPUTE_LOST_AND_SERVICE_CHARGED_EMAIL_TEMPLATE_ID:-pay-notify-stripe-dispute-lost-email-template-id}
```